src/components/RelatedDocumentsList/RelatedDocumentsListPresentation.js

```diff
@@ -5,14 +5,10 @@ import { Link } from 'react-router'
 // FPCC
 import getIcon from 'common/utils/getIcon'
 
-function RelatedDocumentsListPresentation({
-  documents,
-  labelStyling = 'text-left font-medium text-lg uppercase text-charcoal-900',
-}) {
+function RelatedDocumentsListPresentation({ documents }) {
```

Comment:
Just need to remove labelStyling from propTypes as well and this is good to go :)

Reply:
Ah nice catch, will do!